General map/code additions

DoctorOctopusTheOctopi opened the pull request:
- Revamped ODST Locker Room
- Added ODST Vendors
- Added SecTech vendors to the UNSC Frigate's Brig
- Added more loadout options when creating a character.
- Changed auto-CT vote to call at two hours instead of three.
- Removed secret from the gamemode vote.
- Replaced the NT Security Vest in Naval Security Officers lockers with a NSO Vest.

DoctorOctopusTheOctopi committed:
Merge pull request #1 from HaloSpaceStation/alpha

Local Repo Merge 3-23-2018

DoctorOctopusTheOctopi committed:
Merge pull request #2 from HaloSpaceStation/alpha

Local Repo Update 3-24-2018.. Berry good.

DoctorOctopusTheOctopi committed:
Adds ODST Vendors

Made the ODST Vendors only accessable by ODST because people kept stealing their stuff. Thanks Joe for the idea.

DoctorOctopusTheOctopi committed:
Adds SecTech vendors to UNSC Frigate's Brig

They couldn't even get handcuffs, so I gave them vendors.

DoctorOctopusTheOctopi committed:
Player Loadout Additions

Beefs up the loadout system for that Juicy unique RP..

DoctorOctopusTheOctopi committed:
Auto CT Vote time change

Sets the Auto CT Vote to call at two hours instead of three.

DoctorOctopusTheOctopi committed:
Remove secret from being voted.

Changelings shouldn't be showing up on UNSC Frigates and murdering everyone, nor should wizards come in and blow up the Colony..

X0-11 commented on maps/torch/loadout/loadout_accessories.dm:
```diff
@@ -253,14 +191,14 @@
-	display_name = "SolGov major award selection"
-	description = "A medal or ribbon awarded to SolGov personnel for significant accomplishments."
+	display_name = "UNSC major award selection"
+	description = "A medal or ribbon awarded to UNSC personnel for significant accomplishments."
```
These award medals will need their description and name changed on the item itself as-well.

DoctorOctopusTheOctopi replied:
Addressed

X0-11 commented on code/game/machinery/vending.dm:
```diff
@@ -1099,3 +1099,35 @@
+	name = "Armtech 5530"
+	desc = "Cold, dark, smiling with a twisted grin, and slightly depressed. Basically an ODST in vending machine form."
+	product_ads = "Life is woe;Suspect, Investigate, Terminate;CAUTION SHIP SELF DESTRUCT ACTIVATED! Just kidding."
+	icon_state = "Theater"
```
Does the theater vendor fit this?

DoctorOctopusTheOctopi replied:
Its just a placeholder sprite right now, and I didn't want people to get confused with it being a shell vendor so I made it look unique.

X0-11 replied:
Recoloring the current vendors black would work better as a placeholder than the theater vendor

DoctorOctopusTheOctopi replied:
Addressed

DoctorOctopusTheOctopi committed:
Adresses XOs Points

Changed vending machine sprite/color
Changed accessory names.
Cut Nanotrasen Medals

DoctorOctopusTheOctopi commented:
Removing DNM tag as the reason it was added will not happen. I'll just create a separate branch for the modifications that will make it DNM as these are some needed updates and if a merge happens tonight it will leave a lot of quality of life changes unmerged.

DoctorOctopusTheOctopi committed:
Adds the MAA Vest to MAA Lockers

No longer will MAAs be walking around with big bold "NT" on them, no. They will have their proper "MAA" in full display.

DoctorOctopusTheOctopi committed:
Changelog

caelaislinn commented on maps/torch/loadout/loadout_gloves.dm:
```diff
@@ -72,13 +72,13 @@
 	display_name = "gloves, dress"
 	path = /obj/item/clothing/gloves/color/white
-	allowed_roles = MILITARY_ROLES
+
```
In this case it may be better just to remove the line entirely.

caelaislinn commented on maps/torch/loadout/loadout_eyes.dm:
```diff
@@ -23,56 +22,6 @@
-/datum/gear/eyes/medical/prescription
-	display_name = "Medical HUD, prescription"
-	path = /obj/item/clothing/glasses/hud/health/prescription
-	allowed_roles = MEDICAL_ROLES
```
I'd like to bring back job specific HUDs at some point. Right now it's not a priority

caelaislinn reviewed (approved):
Looks good. Welcome to the team
